lib/runtime_options.go

```diff
@@ -27,6 +27,9 @@ type RuntimeOptions struct {
 	// Whether to pass the actual system environment variables to the JS runtime
 	IncludeSystemEnvVars null.Bool `json:"includeSystemEnvVars" envconfig:"include_system_env_vars"`
 
+	// JS compatibility mode: "es6" (Goja+Babel+core.js) or "es51" (plain Goja)
+	CompatibilityMode null.String `json:"compatibilityMode" envconfig:"compatibility_mode"`
+
 	// Environment variables passed onto the runner
 	Env map[string]string `json:"env" envconfig:"env"`
 }
```

Comment:
As far as I can see the `envconfig` here is totally useless ?

Reply:
Indeed, but so are all others? Not sure, but OK, I'll remove it from here.

Reply:
Resolved by 1986149d.